extensions/BugModal/web/bug_modal.js

```diff
@@ -200,7 +200,7 @@ $(function() {
 
     function restoreSavedBugComment() {
         expireSavedComments();
-        let value = JSON.parse(localStorage.getItem(bugCommentCacheKey));
+        let value = Bugzilla.Storage.get(bugCommentCacheKey);
```

Comment:
This line lost the JSON parsing, is that expected?

Reply:
Bugzilla.Storage.get() performs the JSON.parse itself so bug_modal.js doesn't need to do so anymore.